packages/e2e-tests/testFixtureCloneRecording.ts

```diff
@@ -72,6 +76,9 @@ const testWithCloneRecording = base.extend<TestIsolatedRecordingFixture>({
       if (newRecordingId) {
         await deleteTestRecording(newRecordingId);
       }
+      if (testUser && testScope) {
+        await resetTestUser(testUser, testScope);
```

Comment:
Should we log a warning if a `testUser` is supplied without a `testScope`?

Or maybe better yet, can we update the type so it that it requires neither or both?

```ts
type TestIsolatedRecordingFixture = {
  exampleKey: TestRecordingKey;
  pageWithMeta: {
    page: Page;
    recordingId: string;
  };
};

type ScopedTestIsolatedRecordingFixture = TestIsolatedRecordingFixture & {
  testScope: string;
  testUser: string;
};

const testWithCloneRecording = base.extend<
  TestIsolatedRecordingFixture | ScopedTestIsolatedRecordingFixture
>({
```

That way we won't accidentally add any un-scoped authenticated tests?